packages/babel-plugin-transform-remove-console/src/index.js

```diff
@@ -3,6 +3,18 @@ export default function () {
     visitor: {
       CallExpression(path) {
         if (path.get("callee").matchesPattern("console", true)) {
+          const statementPath = path.parentPath.parentPath;
```

Comment:
Does path.parentPath always exist? I assume even when the script is just `debugger`, the parent would be program or file. I just want to make sure that we never get "can't get property parentPath on undefined"